packages/cosmic-proto/test/test-endo.js

```diff
@@ -0,0 +1,42 @@
+// @ts-check
```

Comment:
I'm not sure how valuable these tests are. Keep or remove?

Reply:
yeah, remove. I think they're covered separately. there's some value in having a placeholder for more tests but I don't think "Endo" should have a test file. It should all work in Endo/SES.